pymbe/expansion.py

```diff
@@ -1212,8 +1216,7 @@ def _mbe(
         if mpi.global_master:
             # read restart files
             rst_read = is_file("mbe_tup_idx", self.order) and is_file(
-                "mbe_tup", self.order
-            )
+                "mbe_tup", self.order) and is_file("mbe_occ_orb_contrib", self.order)
```

Comment:
This needs to be changed back, this just determines whether the restart files should be read in and has nothing to do with your changes.
```suggestion
                rst_read = is_file("mbe_tup_idx", self.order) and is_file(
                "mbe_tup", self.order
            )
```